### DISCUSSION TITLE
unit starts blinking and not showing up after 1 day

### DISCUSSION THREAD

**Discussion by stagiaire4** [revk/xESP32-Faikout#235] [General]
We installed a Wi-Fi module called Faikin inside our Daikin air conditioner. Initially, it worked great. We accessed it through its IP address website and successfully integrated it with our home assistant. However, after a day, the Faikin module started blinking a white/blue LED, and now it doesn't show up in the home assistant interface. Additionally, we can't turn it on using the IP website anymore.

![image](https://github.com/revk/ESP32-Faikin/assets/157493563/4ed191ad-ef00-4c64-bb72-bfb9b9db82f8)

![image](https://github.com/revk/ESP32-Faikin/assets/157493563/1e629d86-4904-4fa3-9ca7-e1d6c9dbbec4)

**Comment by revk** [comment]
It says "system off line", I would check the plug/connectors are in place cleanly. Obviously try turning everything off and back on again anyway.

There are some other test that can be done which need access to the MQTT server for logging.

Also, can you do a close up picture of the Faikin module both sides as well, so I can check which model it is.
